src/qibolab/platform.py

```diff
@@ -356,6 +356,34 @@ def execute_pulse_sequence(self, sequence, options, **kwargs):
                     result = new_result
         return result
 
+    def execute_pulse_sequences(self, sequence, options, **kwargs):
+        """Executes a List of PulseSequence.
+
+        Args:
+            sequence (List[:class:`qibolab.pulses.PulseSequence`]): Pulse sequences to execute.
+            options (:class:`qibolab.platforms.platform.ExecutionParameters`): Object holding the execution options.
+            **kwargs: May need them for something
+
+        Returns:
+            Readout results acquired by after execution.
```

Comment:
If you use type hints these lines can be removed.

Reply:
Circular import with `ExecutionParameters` we may want to move them to `executionparameters.py` or something instead of being on the `__init__.py`

Reply:
The import in the `__init__.py` was not used so I could remove it

Reply:
@andrea-pasquale are you sure that `Args:` are not needed in the docstring? How will sphinx know the descriptions of each parameter (eg. "options: Object holding the execution options") just from the type hint if there is no docstring?

Reply:
I see your point. Actually I don't know what should be the standard option.
Usually I prefer to use directly type hints. However, in cases such as this one, or more in general when using sphinx we should take care of docstrings.

Reply:
So should I leave it ?

Reply:
Yes, I would say for this particular case leave both docstrings and type hints.